iot-e2e-tests/common/src/test/java/tests/integration/com/microsoft/azure/sdk/iot/iothub/methods/DirectMethodsTests.java

```diff
@@ -330,4 +332,26 @@ public void invokeMethodWithServiceSideProxy() throws Exception
             proxyServer.stop();
         }
     }
+
+    @Test
+    @StandardTierHubOnlyTest
+    public void invokeMethodWithPayloadAsString() throws Exception
+    {
+        super.SubscribeToMethodAndReceiveAsJsonString();
```

Comment:
```suggestion
        super.subscribeToMethodAndReceiveAsJsonString();
```

method names should start with lower case letters